scst/include/backport.h

```diff
@@ -1474,7 +1474,8 @@ static inline void scsi_build_sense(struct scsi_cmnd *scmd, int desc,
 	 RHEL_RELEASE_CODE -0 < RHEL_RELEASE_VERSION(9, 1))
 
 #if (!defined(RHEL_RELEASE_CODE) || \
-	RHEL_RELEASE_CODE -0 != RHEL_RELEASE_VERSION(8, 7))
```

Comment:
Could you just change the condition from `!=` to `<` instead of duplicating this line for version 8.8.

Reply:
That was going to be my original change (and it worked in 8.8), but I was worried that it was too greedy since the original was as specific as it was. I've changed it now.

Reply:
> I was worried that it was too greedy since the original was as specific as it was

It was my mistake originally, so don't worry about it.